coramin/relaxations/auto_relax.py

```diff
@@ -121,7 +121,7 @@ def _relax_leaf_to_root_ProductExpression(node, values, aux_var_map, degree_map,
         arg2 = replace_sub_expression_with_aux_var(arg2, parent_block)
         relaxation_side = relaxation_side_map[node]
         relaxation = PWMcCormickRelaxation()
-        relaxation.set_input(x=arg1, y=arg2, w=_aux_var, relaxation_side=relaxation_side)
+        relaxation.set_input(x1=arg1, x2=arg2, aux_var=_aux_var, relaxation_side=relaxation_side)
```

Comment:
Style question: why make the x's explicit args and not just take a single arg that is a tuple for multivariate relaxations?  Something like:
```
relaxation.set_input(x=(arg1, arg2), aux_var=_aux_var, relaxation_side=relaxation_side)
```

Reply:
@jsiirola Good question. I don't have much of a reason. In fact, the `MultivariateRelaxation` does almost exactly what you suggested. The only reason we may not want to do that for McCormick relaxations is clear control over which variable gets partitioned (if piecewise refinement is needed). Although, with the tuple notation, we could just say it is the first element of the tuple is the one that gets partitioned. I think being consistent with the `Multivariate` relaxation is probably more important, so I will make this change.

Reply:
@jsiirola I think I prefer having `x1` and `x2` as separate arguments just so it is clear/explicit that there should only be 2 variables involved. Do you feel strongly about making this a single argument which is a tuple or list?